services/prng_generate.proto

```diff
@@ -28,7 +28,7 @@ option java_multiple_files = true;
 /**
  * Generates a pseudorandom number
  */
-message RandomGenerateTransactionBody {
+message PrngGenerateTransactionBody {
```

Comment:
Isn't it redundant to say "Pseudorandom Number Generator Generate Transaction"?

Reply:
True, in a recent call there was a consensus on this name since SDK wanted to use this name.
So it could be `PRNGenerate` ?

Reply:
True, in a recent call there was a consensus on this name since SDK wanted to use this name.
So it could be `PRNGenerate` ?

Reply:
Would've liked to have been on that call. I would think it would be `PseudorandomNumberGeneratorTransaction` or `PrngTransaction`.

Reply:
I understand it to be `PrngTransaction`

Reply:
Yes, updating it to `PrngTransaction` and HederaFunctionality to `PRNG`